timm/models/resnet.py

```diff
@@ -318,7 +334,7 @@ class Bottleneck(nn.Module):
 
     def __init__(self, inplanes, planes, stride=1, downsample=None, cardinality=1, base_width=64,
                  reduce_first=1, dilation=1, first_dilation=None, act_layer=nn.ReLU, norm_layer=nn.BatchNorm2d,
-                 attn_layer=None, aa_layer=None, drop_block=None, drop_path=None):
+                 attn_layer=None, aa_layer=None, drop_block=None, drop_path=None, **kwargs):
```

Comment:
@rwightman FYI this is change that you might want to review specifically. I have added `**kwargs` to `Bottleneck` block that get passed to attention layers. 

This is to pass in `reduction_ratio=0.25` for `se` layers as mentioned in the paper.